docs/ja/installation.md

```diff
@@ -0,0 +1,39 @@
+# インストール
+
+## 直接ダウンロード
+
+[dist フォルダ](https://github.com/vuejs/vue-validator/tree/dev/dist)を参照してください。dist フォルダにあるファイルは常に最新の安定版であることに注意してください。 - `dev` ブランチのソースは最新ではありません。
+
+## CDN
+
+### jsdelivr
+
+```html
+<script src="https://cdn.jsdelivr.net/vue.validator/2.1.1/vue-validator.min.js"></script>
+```
+
+## NPM
+
+### 安定版
+
+    $ npm install vue-validator
+
+### 開発版
+
+    $ git clone https://github.com/vuejs/vue-validator.git node_modules/vue-validator
+    $ cd node_modules/vue-validator
+    $ npm install
+    $ npm run build
+
+CommonJS で使用されるとき、明示的に `Vue.use()` 経由でバリデーターをインストールしなければなりません:
+
+> :warning: もし `vue-router` を使用している場合は、 `router#start` の前に `Vue.use()` でインストールしなければなりません。
```

Comment:
When I check on my localhost, this warning won't be complied as image. Why...

### current on my localhost

![image](https://cloud.githubusercontent.com/assets/1726225/15043190/7f0c3ac6-1305-11e6-933f-3d00e871db5b.png)

### expected 

![image](https://cloud.githubusercontent.com/assets/1726225/15043212/bfa915a4-1305-11e6-82d1-079582713d75.png)